src/components/Header/AddMovieButton/AddMovieButton.tsx

```diff
@@ -1,7 +1,22 @@
 import styles from "./addMovieButton.module.css";
+import AddModal from "../../Modal/AddModal";
+import { useState } from "react";
 
 function AddMovieButton() {
```

Comment:
It would be better give this button less specific name (you can take some naming ideas from css frameworks ) 

and store this component in `common` folder